lib/src/Base/Common/openturns/Study.hxx

```diff
@@ -67,6 +67,15 @@ public:
    */
   Study();
 
+#if defined OPENTURNS_HAVE_LIBXML2
+
+  /**
+   * Parameter constructor
+   */
+  Study(const String & fileName);
+
+#endif /* OPENTURNS_HAVE_LIBXML2 */
+
```

Comment:
OPENTURNS_HAVE_XXX macros are unavailable when compiling outside the library, hence I suggest to not use it in the header ; leave the constructor unconditionally in the header and use the macro inside the constructor code.